content/docs/static-type-checking.md

```diff
@@ -162,6 +285,12 @@ Note that it is a **third party** project, and is not a part of Create React App
 
 You can also try [typescript-react-starter](https://github.com/Microsoft/TypeScript-React-Starter#typescript-react-starter).
 
+Ready to code you are! We recommend to check out the following resources to learn more about Typescript:
```

Comment:
Suggest changing "Ready to code you are!" to "You are now ready to code!"

Reply:
Ha, for some reason I imagined sounding like an old English knight :D

Reply:
😆  I like the playful tone. I just wanted to avoid potentially confusing anyone reading the docs who might not be a native speaker. (Even though the chance of confusion was probably pretty small here.)

Reply:
Ha, good looking out.